src/Transports/MassTransit.ActiveMqTransport/Topology/Topologies/ActiveMqMessageConsumeTopology.cs

```diff
@@ -16,16 +18,15 @@ public class ActiveMqMessageConsumeTopology<TMessage> :
         IActiveMqMessageConsumeTopologyConfigurator
         where TMessage : class
     {
-        readonly string _consumerName;
+        //readonly string _consumerName;
         readonly IActiveMqMessagePublishTopology<TMessage> _publishTopology;
+        private readonly ActiveMqConsumeTopology _consumeTopology;
         readonly IList<IActiveMqConsumeTopologySpecification> _specifications;
 
-        public ActiveMqMessageConsumeTopology(IActiveMqMessagePublishTopology<TMessage> publishTopology)
+        public ActiveMqMessageConsumeTopology(IActiveMqMessagePublishTopology<TMessage> publishTopology, ActiveMqConsumeTopology consumeTopology)

```

Comment:
should not depend upon concrete types

Reply:
new commits have different implementation